packages/node/CHANGELOG.md

```diff
@@ -6,7 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-
+## [0.33.0] - 2022-04-06
+### Added
+- Add support for handle manifest 1.0.0 (#845)
```

Comment:
Can you add a link to manifest 1.0.0 docs?

Reply:
We don't have document for manifest 1.0.0 yet, we can update it once available